core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -859,25 +884,68 @@ public void removeBlocks(Collection<Long> blockIds, boolean delete) throws Unava
   }
 
   @Override
-  public boolean isNotServing(long workerId) {
-    return mDecommissionedWorkers.getFirstByField(ID_INDEX, workerId) != null;
+  public boolean isRejected(WorkerNetAddress address) {
+    return mRejectWorkers.contains(address);
   }
 
   @Override
-  public void decommissionWorker(String workerHostName)
+  public void decommissionWorker(DecommissionWorkerPOptions requestOptions)
```

Comment:
can we cover the states of the worker in question and the expected outcome in unit tests? e.g. if the worker is in the lost worker set, then decommission should succeed and the worker will be moved from the lost set to the decommissioned set.

Reply:
sgtm

Reply:
one UT added